WooCommerce/WooCommerceTests/Tools/DeveloperEmailCheckerTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+@testable import WooCommerce
```

Comment:
Nice! 10_000 extra points for adding tests!

Reply:
Nice! I want to write unit tests too. I hope you can suggest things I should test :)